metrics_utility/automation_controller_billing/helpers.py

```diff
@@ -1,9 +1,123 @@
 import json
 
 from itertools import chain
+from typing import Dict, Optional
 
 import pandas as pd
 
+from django.db import connection
+from django.utils.dateparse import parse_datetime
+
+from metrics_utility.logger import logger
+
+
+def get_last_entries_from_db() -> Optional[str]:
+    """
+    Get AUTOMATION_ANALYTICS_LAST_ENTRIES directly from database.
+
+    Returns:
+        Optional[str]: JSON string from database, or None if not found or error occurs
+    """
+    try:
+        with connection.cursor() as cursor:
+            cursor.execute("""
+                SELECT value
+                FROM conf_setting
+                WHERE key = 'AUTOMATION_ANALYTICS_LAST_ENTRIES'
+                LIMIT 1
+            """)
+            result = cursor.fetchone()
+
+            if result and result[0]:
+                return result[0]  # This is the JSON value
+            else:
+                return None
+    except Exception as e:
+        logger.error(f'Error getting AUTOMATION_ANALYTICS_LAST_ENTRIES from database: {e}')
+    return None
+
+
+def get_license_info_from_db() -> Dict[str, str]:
+    """Get license information directly from the database."""
+    license_info = {}
+    try:
+        with connection.cursor() as cursor:
+            cursor.execute("""
+                SELECT key, value
+                FROM conf_setting
+                WHERE key IN (
+                    'PLATFORM', 'INSTALL_UUID', 'TOWER_BASE_URL', 'CONTROLLER_VERSION', 'LICENSE', 'SUBSCRIPTION_NAME', 'SKU',
+                    'SUPPORT_LEVEL', 'USAGE', 'PRODUCT_NAME', 'VALID_KEY',
+                    'SATELLITE', 'POOL_ID', 'SUBSCRIPTION_ID', 'ACCOUNT_NUMBER',
+                    'CURRENT_INSTANCES', 'AUTOMATED_INSTANCES', 'AUTOMATED_SINCE',
+                    'TRIAL', 'GRACE_PERIOD_REMAINING', 'COMPLIANT',
+                    'DATE_WARNING', 'DATE_EXPIRED', 'SUBSCRIPTION_USAGE_MODEL',
+                    'FREE_INSTANCES', 'INSTANCE_COUNT'
```

Comment:
```suggestion
                    'INSTALL_UUID', 'TOWER_BASE_URL', 'LICENSE', 'SUBSCRIPTION_USAGE_MODEL'
```

I think we only use these now, as most of the keys would come from inside the license

(though, if there IS a `CONTROLLER_VERSION`, maybe `get_controller_version_from_db` should also use that?)